components/genesys-email-flow/README.md

```diff
@@ -0,0 +1,111 @@
+We are now in the last stage of setting up this blueprint. In this last step, we are going to use Genesys Cloud `CX as Code`, the Genesys Cloud Python SDK and Genesys Cloud's Archy tools to deploy all of the Genesys Cloud objects used to handle the email flow in this blueprint. 
+
+# Pre-Requisites
+
+1. **Install Terraform**. Install the latest Terraform binary. Instructions for installing Terraform can be found [here](https://www.terraform.io/downloads.html). 
+2. **Install Archy**. Install the latest Genesys Cloud Archy import/export tool. Instructions for installing Archy can be found [here](https://developer.genesys.cloud/devapps/archy/).
+3. **Install Python 3.7**. This Terraform flow wrappers an Archy call using Python 3.7+. Please make sure you have Python installed.
+4. **Install the Genesys Cloud python SDK**. The python script that calls Archy also uses our Genesys Cloud python SDK. Instructions for installing the Genesys Cloud python SDK can be found [here](https://developer.genesys.cloud/api/rest/client-libraries/python/).
+
+# Deployment Steps
+Once the pre-requisite tools are installed you need to take the following actions to deploy your flow.
+
+1. **Set your credentials and AWS region**. 
+2. **Setup your `CX as Code` flow.auto.tfvars file**.
+3. **Configure your Terraform backend**.
+4. **Initialize Terraform**.
+5. **Apply your Terraform changes**.
+
+## Setup your credentials and AWS Regions. 
+All Genesys Cloud OAuth2 credentials and AWS region configuration used by `CX as Code`is handled through environment variables. 
+The following environment variables need to be set before you run your Terraform configuration
+
+1. `GENESYSCLOUD_OAUTHCLIENT_ID`. The OAuth2 Client id of the Genesys Cloud OAuth2 client credential grant the Genesys Cloud `Cx as Code` provider will run under. Information on how to setup a Genesys Cloud OAuth2 client credential grant can be found [here](https://help.mypurecloud.com/articles/create-an-oauth-client/).
+
+2. `GENESYSCLOUD_OAUTHCLIENT_SECRET`. The OAuth2 secret of the Genesys Cloud OAuth2 client credential grant the Genesys Cloud `Cx s Code` provider will run under.
+
+3. `GENESYSCLOUD_REGION`. The region used by the Genesys Cloud OAuth2 client. Valid values can be found in the `AWS region` field listed [here](https://developer.genesys.cloud/api/rest/).
+
+4. `GENESYSCLOUD_API_REGION`. The Genesys Cloud API endpoint the Genesys Cloud SDK will connect with. Valid values can be found in the `Apps URL` field listed [here](https://developer.genesys.cloud/api/rest/).
```

Comment:
The API Urls are under `API Server` field instead of `Apps URL` field.

Reply:
Nice.  Good catch.